libnymea/interfaces/thermostat.json

```diff
@@ -1,5 +1,5 @@
 {
-    "description": "The thermostat interface describes devices which have a target temperature value and regulate themselves to match that target temperature.",
+    "description": "The thermostat interface describes devices which have a target temperature value and regulate themselves to match that target temperature. A thermostat may support detecting for open windows in which case the windowOpenDetected state should be implemented. Otoh, a thermostat may allow setting a window open lock in which case the windowOpen state should be implemented.",
```

Comment:
`Otoh`, a thermostat...
Did you mean `Although`?

Reply:
https://www.urbandictionary.com/define.php?term=OTOH

Otoh, probably shouldn't use that sort of slang in docs :D